pkg/shell/cockpit-networking.js

```diff
@@ -2700,10 +2700,11 @@ PageNetworkBondSettings.prototype = {
             }
         }
 
+        /* TODO: Migrate away from jQuery spaggheti */
```

Comment:
possibly add comment about adding linking code between labels and input controls when this is cleaned up